Source/relay/TourGuide/Quests/8-bit Realm.ash

```diff
@@ -148,14 +148,15 @@ void Q8bitRealmGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [
     }
 
     // Figure out if the user is better-suited to adventure elsewhere.
-    string highestPointColor;
+    string highestPointColor = currentColor;
```

Comment:
yeah this is the right initialization and solves that issue pretty well. thank you!